content/en/blog/_posts/2025-02-14-cloud-controller-manager-chicken-egg-problem.md

```diff
@@ -189,6 +188,8 @@ spec:
     metadata:
       labels:
         app.kubernetes.io/name: cloud-controller-manager
+      annotations:
+        kubernetes.io/description: Cloud controller manager for my infrastructure
```

Comment:
I can see the case for using double quotes here; I think the example is more readable without them.

Reply:
ack, thanks for catching this!